mlrun/api/launcher.py

```diff
@@ -199,3 +195,7 @@ def _store_function(
                 struct, runtime.metadata.name, runtime.metadata.project, versioned=True
             )
             run.spec.function = runtime._function_uri(hash_key=hash_key)
+
+    def _refresh_function_metadata(self, runtime: "mlrun.runtimes.BaseRuntime"):
+        """metadata refresh is not required in the API"""
```

Comment:
so why is it under `api`. improve documentation
"this is just a stub, doing nothing as it is not required / needed by the api"